android/build.gradle

```diff
@@ -81,69 +81,31 @@ android {
     targetCompatibility JavaVersion.VERSION_1_8
   }
 
-  lintOptions{
-    abortOnError false
-    disable 'GradleCompatible'
-  }
-
   externalNativeBuild {
      cmake {
           path "./src/main/rnmmkv/CMakeLists.txt"
       }
   }
-
-  packagingOptions {
-    excludes = ["**/libc++_shared.so","**/libjsi.so","**/libreactnativejni.so", "META-INF/MANIFEST.MF"]
-  }
-}
-
-repositories {
-  mavenCentral()
-  mavenLocal()
-  google()  
-
-  if (reactNativeDir.exists()) {
-    maven {
-      url reactNativeDir.toString()
-      name androidSourcesName
-    }
-    logger.info(":${project.name}:reactNativeAndroidRoot ${reactNativeDir.canonicalPath}")
-  }
-
-  if (!found) {
-    throw new GradleException(
-            "${project.name}: unable to locate React Native android sources. " +
-                    "Ensure you have you installed React Native as a dependency in your project and try again."
-    )
-  }
- 
 }
 
 dependencies {
   //noinspection GradleDynamicVersion
   implementation 'com.facebook.react:react-native:+'
```

Comment:
This should be `implementation("com.facebook.react:react-android")`

Reply:
Nope, both works according to [RN docs](https://github.com/facebook/react-native/tree/main/android), however, I'm gonna change it because we are dropping support for old RN versions anyways.